solutions/07-regional-tables/movr/pricing/data/pricing_database.sql

```diff
@@ -16,7 +16,7 @@ CREATE TABLE movr_pricing.vip_rates (
     rate_code STRING PRIMARY KEY,
     market STRING NOT NULL,
     description STRING NOT NULL, 
-    creation_time TIMESTAMP NOT NULL, 
+    creation_time TIMESTAMP NOT NULL 
 );
 
 ALTER TABLE movr_pricing.promo_codes SET locality GLOBAL;
```

Comment:
This does not contain the solution to ex 07 (which involves setting locality of vip_rates)

Reply:
added solution